src/useValidation/useValidation.ts

```diff
@@ -29,6 +31,10 @@ export function useValidation<T>(
   const valid = computed(() => error.value === '')
   const invalid = computed(() => !valid.value)
   const pending = ref(false)
+  const initialValue = unref(value)
```

Comment:
What if it's an object or an array? 

Also validation doesn't happen unless mounted. But the value can change before then. So this won't match expectation in all cases